staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1281,6 +1419,71 @@ func (c *cacheWatcher) nextBookmarkTime(now time.Time, bookmarkFrequency time.Du
 	return heartbeatTime, true
 }
 
+// wasBookmarkAfterResourceVersionReceivedLocked same as wasBookmarkAfterResourceVersionReceived just acquires a lock
+func (c *cacheWatcher) wasBookmarkAfterResourceVersionReceivedLocked() bool {
```

Comment:
Remove `Locked` - we use `Locked` in the name when the lock is already held when this function is being called.

This also means updating names of the functions below.